urdfenvs/urdfCommon/generic_robot.py

```diff
@@ -65,18 +62,24 @@ def read_limits(self) -> None:
     def set_acceleration_limits(self):
         pass
 
-    def get_indexed_joint_info(self) -> dict:
+    def get_indexed_joint_info(self) -> None:
         """Get indexed joint info.
 
         This function can be used for debugging and finding
         the correct joint indices for self.setJointIndices.
```

Comment:
This function is now central, so the comment on debugging and finding the correct joint indices should be modified.